guides/authn_authz/scopes.md

```diff
@@ -202,6 +202,10 @@ In this example, the scope is called `user` and it is the `default` scope that i
 
 * `access_path` - a list of keys that define the path to the identifying field in the scope struct. The generators generate code like `where: schema_key == ^scope.user.id`.
 
+* `route_prefix` - a path template string for how resources should be nested. For example, `/orgs/:org` would generate routes like `/orgs/:org/posts`. The parameter segment (`:org`) will be replaced with the appropriate scope access value in templates and LiveViews.
+
+* `route_access_path` - a list of keys that define the path to the field used in route generation. This is particularly useful for user-friendly URLs where you might want to use a slug instead of an ID. If not specified, it defaults to `access_path`. For example, `[:organization, :slug]` would generate URLs using organization slugs instead of IDs.
```

Comment:
Do we need this? Can't we rely on Phoenix.Param?

Reply:
If someone has phoenix param implemented, they can just use `[:organization]` as parameter. But we must have some path, we cannot assume how the scope struct looks.